src/matcha_ml/cli/_validation.py

```diff
@@ -139,6 +139,9 @@ def region_typer_callback(region: str) -> str:
     Returns:
         str: the region after checks are passed.
     """
+    if not region:
```

Comment:
Do you prefer this or:

```
if region:

    try:
        region_validation(region)
    except MatchaInputError as e:
        raise BadParameter(str(e))

return region
```

Reply:
`not region` check is a bit more specific in my opinion (we specifically check for the empty string, as we know that it maybe empty when no CLI option is passed -- so we do not check the rest --, and it cannot be empty after the prompt)

Reply:
Sounds good to me.